INSTALL.md

```diff
@@ -30,6 +30,9 @@ The recommended approach for using **al-folio** is to first create your own site
 5. Finally, in the repository page go to `Settings -> Pages -> Build and deployment`, make sure that `Source` is set to `Deploy from a branch` and set the branch to `gh-pages` (NOT to master).
 6. Wait until the GitHub actions finish (check your repository **Actions** tab), then simply navigate to `https://<your-github-username>.github.io` in your browser. At this point you should see a copy of the theme's [demo website](https://alshedivat.github.io/al-folio/).
 
+In case you face the error: "Input required and not supplied: token" in the Lighthouse Badger action, follow the instructions below:
```

Comment:
We actually have a [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#when-i-manually-run-the-lighthouse-badger-workflow-it-fails-with-error-input-required-and-not-supplied-token-how-do-i-fix-that) entry for this. I don't think this is needed in the INSTALL section, since this is an optional feature. Maybe we can add a section for optional features in the `CUSTIME.md`?

Reply:
@george-gca Sure, I've updated it.  I've moved this optional feature details to a new section in CUSTOMIZE.md